ApiCustomer/CostumerApi/CustomerApi.csproj

```
@@ -14,6 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\AppModels\AppModels.csproj" />
     <ProjectReference Include="..\AppServices\AppServices.csproj" />
```

A camada da API deve ter como referência apenas os serviços da camada de aplicação
```suggestion
  <ItemGroup>
    <ProjectReference Include="..\AppServices\AppServices.csproj" />
```